pkg/apis/direct.csi.min.io/v1beta3/conversion_generated.go

```diff
@@ -0,0 +1,407 @@
+	out.ModelNumber = in.ModelNumber
+	out.SerialNumber = in.SerialNumber
+	out.TotalCapacity = in.TotalCapacity
+	out.PhysicalBlockSize = in.PhysicalBlockSize
+	out.LogicalBlockSize = in.LogicalBlockSize
+	out.Topology = *(*map[string]string)(unsafe.Pointer(&in.Topology))
+	out.AccessTier = v1beta2.AccessTier(in.AccessTier)
+	// INFO: in.FilesystemUUID opted out of conversion generation
+	// INFO: in.PartitionUUID opted out of conversion generation
+	// INFO: in.MajorNumber opted out of conversion generation
+	// INFO: in.MinorNumber opted out of conversion generation
+	// INFO: in.UeventSerial opted out of conversion generation
+	// INFO: in.UeventFSUUID opted out of conversion generation
+	// INFO: in.WWID opted out of conversion generation
+	// INFO: in.Vendor opted out of conversion generation
+	// INFO: in.DMName opted out of conversion generation
+	// INFO: in.DMUUID opted out of conversion generation
+	// INFO: in.MDUUID opted out of conversion generation
+	// INFO: in.PartTableUUID opted out of conversion generation
+	// INFO: in.PartTableType opted out of conversion generation
+	// INFO: in.Virtual opted out of conversion generation
+	// INFO: in.ReadOnly opted out of conversion generation
+	// INFO: in.Partitioned opted out of conversion generation
+	// INFO: in.SwapOn opted out of conversion generation
+	// INFO: in.Master opted out of conversion generation
+	out.Conditions = *(*[]v1.Condition)(unsafe.Pointer(&in.Conditions))
+	return nil
+}
+
+// Convert_v1beta3_DirectCSIDriveStatus_To_v1beta2_DirectCSIDriveStatus is an autogenerated conversion function.
+func Convert_v1beta3_DirectCSIDriveStatus_To_v1beta2_DirectCSIDriveStatus(in *DirectCSIDriveStatus, out *v1beta2.DirectCSIDriveStatus, s conversion.Scope) error {
+	return autoConvert_v1beta3_DirectCSIDriveStatus_To_v1beta2_DirectCSIDriveStatus(in, out, s)
+}
+
+func autoConvert_v1beta2_DirectCSIDriveStatus_To_v1beta3_DirectCSIDriveStatus(in *v1beta2.DirectCSIDriveStatus, out *DirectCSIDriveStatus, s conversion.Scope) error {
+	out.Path = in.Path
+	out.AllocatedCapacity = in.AllocatedCapacity
+	out.FreeCapacity = in.FreeCapacity
+	out.RootPartition = in.RootPartition
+	out.PartitionNum = in.PartitionNum
+	out.Filesystem = in.Filesystem
+	out.Mountpoint = in.Mountpoint
+	out.MountOptions = *(*[]string)(unsafe.Pointer(&in.MountOptions))
+	out.NodeName = in.NodeName
+	out.DriveStatus = DriveStatus(in.DriveStatus)
+	out.ModelNumber = in.ModelNumber
+	out.SerialNumber = in.SerialNumber
+	out.TotalCapacity = in.TotalCapacity
+	out.PhysicalBlockSize = in.PhysicalBlockSize
+	out.LogicalBlockSize = in.LogicalBlockSize
+	out.Topology = *(*map[string]string)(unsafe.Pointer(&in.Topology))
+	out.AccessTier = AccessTier(in.AccessTier)
+	// INFO: in.FilesystemUUID opted out of conversion generation
+	// INFO: in.PartitionUUID opted out of conversion generation
+	// INFO: in.MajorNumber opted out of conversion generation
+	// INFO: in.MinorNumber opted out of conversion generation
```

Comment:
Looks like these fields are opted out for conversion, We need to remove the `+k8s:conversion-gen=false` comment on these fields in `pkg/apis/direct.csi.min.io/v1beta2/types.go`

Reply:
Done